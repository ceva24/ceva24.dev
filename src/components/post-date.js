import React from "react"

export default ({children}) => (
    <strong class="secondary-description">
        {children}
    </strong>
)
