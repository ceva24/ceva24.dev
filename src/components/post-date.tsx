interface PostDateProps {
    children: React.ReactNode;
}

const PostDate: React.FC<PostDateProps> = ({ children }: PostDateProps) => (
    <strong className="secondary-description">{children}</strong>
);

export { PostDate };
