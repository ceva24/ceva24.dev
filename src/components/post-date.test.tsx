import React from "react";
import { render } from "@testing-library/react";
import { PostDate } from "./post-date";

describe("post date", () => {
    it("renders", () => {
        const date = "2000-01-01";

        const { asFragment } = render(<PostDate>{date}</PostDate>);

        expect(asFragment()).toMatchSnapshot();
    });
});
