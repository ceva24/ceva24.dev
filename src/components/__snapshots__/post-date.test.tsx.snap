// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`post date renders 1`] = `
<DocumentFragment>
  <strong>
    2000-01-01
  </strong>
</DocumentFragment>
`;
